terraform/environments/delius-jitbit/monitoring_lb.tf

```diff
@@ -11,7 +11,7 @@ resource "aws_cloudwatch_metric_alarm" "lb_high_5XX_count" {
   insufficient_data_actions = []
   alarm_actions             = [aws_sns_topic.jitbit_alerting.arn]
   ok_actions                = [aws_sns_topic.jitbit_alerting.arn]
-  treat_missing_data        = "missing"
+  treat_missing_data        = "notBreaching"
```

Comment:
These metrics are push metrics which means they wont be anything to alarm on if there are no such errors